src/lib.rs

```diff
@@ -21,149 +75,262 @@
 //! if rng.gen() { // random bool
 //!     let x: f64 = rng.gen(); // random number in range (0, 1)
 //!     println!("x is: {}", x);
+//!     let char = rng.gen::<char>(); // Sometimes you need type annotation
+//!     println!("char is: {}", char);
 //!     println!("Number from 0 to 9: {}", rng.gen_range(0, 10));
 //! }
 //! ```
 //!
-//! The key function is [`Rng::gen()`]. It is polymorphic and so can be used to
-//! generate many types; the [`Standard`] distribution carries the
-//! implementations. In some cases type annotation is required, e.g.
-//! `rng.gen::<f64>()`.
 //!
-//! # Getting random values
+//! ## Random number generators (RNGs)
+//!
+//! Pulling random numbers out of 'thin air' is very hard for a computer. Not
+//! impossible, but hard and slow. Generally the operating systems collects some
+//! of them, by measuring different kinds of noice/jitter in the hardware.
+//!
+//! What is always used instead is a psuedo-random number generator. This is a
+//! clever algorithm that can produce an infinite stream of random numbers from
```

Comment:
psuedo-random or apparently random; the values are not actually random (by the usual definitions)

Reply:
Typo: psuedo -> pseudo

Reply:
Ok, I'll change it here. But I don't think we should get too pedantic and call it `pseudo-random` everywhere. It is theoretically right, but not an all that interesting distinction.